src/components/LandingText/style.module.scss

```diff
@@ -0,0 +1,96 @@
+@use '../../styles/vars.scss' as vars;
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 1.813rem;
+  position: relative;
+  top: 20%;
+
+  .subheading {
+    @media screen and (max-width: vars.$breakpoint-md) {
+      font-size: 1rem !important;
+    }
+  }
+
+  .header {
+    background: linear-gradient(90deg, #a5e0fb 0%, #dff5ff 100%);
+    background-clip: text;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    @media screen and (max-width: vars.$breakpoint-md) {
+      font-size: 2.75rem !important;
+    }
+  }
+
+  .gradient {
+    background: linear-gradient(
+      90deg,
+      #e981a0 64.06%,
+      #ff6f6f 69.49%,
+      #f9a857 75.8%,
+      #51c0c0 83.16%,
+      #62b0ff 89.12%,
+      #816dff 97.71%
```

Comment:
it seems the rainbow is only applying to "Diego", is that intentional

![image](https://github.com/user-attachments/assets/f16a7cc5-8e6d-4559-a124-f8d449bbcb61)

vs the design:

![image](https://github.com/user-attachments/assets/16229ac5-daa0-4d75-bbce-0b81a82eb16f)

you could probably just adjust these percentages to start at 0 if you want to match the design. also maybe set the `font-weight` to `bold`